### PR TITLE
[MODORDSTOR-418] Add karate tests for updating orders data after inventory ownership changes

### PR DESCRIPTION
**Pull request by Saba-Zedginidze-EPAM**
## Purpose
[[MODORDSTOR-418] Add karate tests for updating orders data after inventory ownership changes
](https://folio-org.atlassian.net/browse/MODORDSTOR-418)

## Approach
- Add new reusables
- Add new tests to cover 3/4 cases

## Screenshots
image
image

### CODE REVIEW

File: acquisitions/src/main/resources/thunderjet/consortia/reusable/verifyOwnership.feature [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  Background:
+    * url baseUrl
+
+  Scenario: Verify that Holding and Item are updated with the new ownership
+    Given path 'holdings-storage/holdings'
+    And param query = 'instanceId==' + instanceId
+    When method GET
+    Then status 200
+    And match response.totalRecords == 1
+    And match response.holdingsRecords[0].id == holdingId
```
Comment (BKadirkhodjaev) [inline acquisitions/src/main/resources/thunderjet/consortia/reusable/verifyOwnership.feature:13]: Can be improved with `match each response.holdingsRecords[*].id == holdingId`

File: acquisitions/src/main/resources/thunderjet/mod-orders/reusable/create-order-line-with-instance.feature [hunk 2]
```diff
@@ -0,0 +1,37 @@
+@ignore
+Feature: Create order line
```
Comment (BKadirkhodjaev) [inline acquisitions/src/main/resources/thunderjet/mod-orders/reusable/create-order-line-with-instance.feature:2]: Did you try reusing/amending with the incoming `instanceId` the existing `createOrderLine` function?

File: acquisitions/src/main/resources/thunderjet/mod-orders/features/update-inventory-ownership-changes-order-data.feature [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+    * def v = call openOrder { orderId: '#(orderId)' }
+
+    Given path 'orders/titles'
+    And param query = 'instanceId==' + instanceId
+    When method GET
+    Then status 200
+    * def titleId = response.titles[0].id
+
+
+  Scenario: Test for changing ownership of Holdings to affect Pieces and PoLines
```
Comment (BKadirkhodjaev) [inline acquisitions/src/main/resources/thunderjet/mod-orders/features/update-inventory-ownership-changes-order-data.feature:48]: A @Positive indicator can be added to both scenarios

File: acquisitions/src/main/resources/thunderjet/consortia/reusable/updateItemOwnership.feature [hunk 4]
```diff
@@ -0,0 +1,18 @@
+Feature: Update Item Ownership
+
+  Background:
+    * url baseUrl
+
+  Scenario: Update Item Ownership
```
Comment (BKadirkhodjaev) [inline acquisitions/src/main/resources/thunderjet/consortia/reusable/updateItemOwnership.feature:6]: In our reusables at the Scenario level we usually go for a different naming pattern, try something like `updateItemOwnership`, I remember Damien telling me this too back in June

File: acquisitions/src/main/resources/thunderjet/mod-orders/reusable/create-order-line-with-instance.feature (hunk 2 above)
Comment (Saba-Zedginidze-EPAM) [reply acquisitions/src/main/resources/thunderjet/mod-orders/reusable/create-order-line-with-instance.feature:2]: Yes and it was producing issues for other tests, that's why I created a different one

File: acquisitions/src/main/resources/thunderjet/mod-orders/features/update-inventory-ownership-changes-order-data.feature [hunk 5]
```diff
@@ -0,0 +1,113 @@
+Feature: Update Item and Holding Ownership Changes Pieces and PoLine Data
```
Comment (azizbekxm) [inline acquisitions/src/main/resources/thunderjet/mod-orders/features/update-inventory-ownership-changes-order-data.feature:1]: I assume this scenario name should be move holding/item

File: acquisitions/src/main/resources/thunderjet/consortia/features/update-inventory-ownership-changes-order-data.feature [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+
+    * configure headers = headersUniversity
+    Given path 'inventory/items-by-holdings-id'
+    And param query = 'holdingsRecordId==' + holdingId
+    When method GET
+    Then status 200
+    * def itemId = response.items[0].id
+
+
+  Scenario: Test for changing ownership of Holdings to affect Pieces and PoLines
```
Comment (azizbekxm) [inline acquisitions/src/main/resources/thunderjet/consortia/features/update-inventory-ownership-changes-order-data.feature:74]: Also need scenario for change ownership of item

**Review by BKadirkhodjaev — APPROVED** [review]

**Review by SerhiiNosko — APPROVED** [review]

**Pull request merged**